en/setup/dev_env_windows_msys.md

```diff
@@ -6,5 +6,5 @@
 
 Download and install the following on your system:
 
-* [PX4 Toolchain Installer v14 for Windows Download](http://firmware.diydrones.com/Tools/PX4-tools/px4_toolchain_installer_v14_win.exe) (32/64 bit systems, complete build system, drivers)
-* [PX4 USB Drivers](https://github.com/mavlink/qgroundcontrol/raw/master/deploy/px4driver.msi) (32/64 bit systems)
+* [PX4 Toolchain Installer v14 for Windows Download](http://firmware.diydrones.com/Tools/PX4-tools/px4_toolchain_installer_v14_win.exe) (32/64 bit systems, complete build system, drivers) 
```

Comment:
@MaEtUgR @bkueng The installer isn't downloading and the PX4 USB drivers are no longer present at the specified location (they changed).

Advice? I propose we simply remove this option altogether since as far as I can tell it hasn't worked for a while and isn't supported.

Reply:
FYI, I am merging this, but would still like advice.